File: Python3/primo.py
"""Problema: Crie um algoritmo que leia um numero inteiro, e diga se ele é um numero primo ou não."""

num = int(input("Digite um número: "))
if num > 1:
   for i in range(2, num):
      if (num % i) == 0:
         print("O número %d não é primo." %(num))
         break
      else:
         print("O número %d é primo." %(num))
         break

File: java/primo.java
import java.util.Scanner;

class primo {
   public static void main(String args[]) {
      Scanner sc = new Scanner(System.in);
      int numero, contador, i;
      contador = 0;
      i = 2;
      System.out.println("Digite um numero : ");
      numero = sc.nextInt();
      while (i < numero) {
         if (numero % i == 0)
            contador = contador + 1;
         i++;
      }
      if (contador > 0)
         System.out.println("O numero " + numero + " nao e primo");
      else
         System.out.println("O numero " + numero + "  e primo");
      sc.close();
   }
}
